ui/components/app/configure-snap-popup/configure-snap-popup.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  BUTTON_VARIANT,
+  Button,
+  Text,
+  Box,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+} from '../../component-library';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  AlignItems,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+
+export enum ConfigureSnapPopupType {
+  CONFIGURE = 'configure',
+  INSTALL = 'install',
+}
+
+export default function ConfigureSnapPopup({
+  type,
+  isOpen,
+  onClose,
+  link,
+}: {
+  type: ConfigureSnapPopupType;
+  isOpen: boolean;
+  onClose: () => void;
+  link: string;
+}) {
+  const t = useI18nContext();
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={onClose}
+      className="whats-new-popup__popover"
```

Comment:
This class name seems wrong?

Reply:
Classname was not used, and has been removed.